druid-shell/src/backend/gtk/window.rs

```diff
@@ -927,7 +940,9 @@ impl WindowState {
                     )
                     .ok()
                     .map(|s| FileInfo {
-                        path: s.into(),
+                        // `get_file_dialog_path` should make sure one and only one
+                        // file is passed
```

Comment:
Let's be confident here :) "`get_file_dialog_path` guarantees that save dialogs only return one path", or something like that.

Reply:
Ok, I'm going to change the comment